lib/ember-app.js

```diff
@@ -8,23 +8,20 @@ var najax = require('najax');
 var debug   = require('debug')('ember-cli-fastboot:ember-app');
 var emberDebug = require('debug')('ember-cli-fastboot:ember');
 
-function EmberApp(options) {
-  this.appFile = options.appFile;
-  this.vendorFile = options.vendorFile;
+var HTMLSerializer = new SimpleDOM.HTMLSerializer(SimpleDOM.voidMap);
 
-  debug("app created; app=%s; vendor=%s", this.appFile, this.vendorFile);
+function EmberApp(options) {
+  var appFilePath = options.appFile;
+  var vendorFilePath = options.vendorFile;
 
-  // Promise that represents the completion of app boot.
-  var appBoot = RSVP.defer();
+  debug("app created; app=%s; vendor=%s", appFilePath, vendorFilePath);
 
   // Create the sandbox, giving it the resolver to resolve once the app
   // has booted.
-  var sandbox = createSandbox(appBoot.resolve, {
-    najax: najax
-  });
+  var sandbox = createSandbox({ najax: najax });
```

Comment:
We should think about exposing a more flexible primitive for this so addons can do similar things.